migrations/etl/2_Mine_manager.sql

```diff
@@ -23,6 +25,168 @@ BEGIN
+    WITH now_manager AS (
+        SELECT
+            SUBSTRING(now_contact.cid from 1 for 7) AS mine_no      ,
+            contact_info.cid                        AS contact_cid  ,
+            contact_info.add_dt                     AS add_dt
+        FROM mms.mmsccc now_contact
+        INNER JOIN mms.mmsccn contact_info on
+            contact_info.cid=now_contact.cid_ccn
+        WHERE
+            SUBSTRING(now_contact.type_ind from 3 for 1) = 'Y'
+    ),
+    --2. Latest NoW with manager attached
+    latest_now AS (
+        SELECT
+            mine_no                 ,
+            MAX(add_dt)        add_dt
+        FROM now_manager
+        GROUP BY
+            mine_no
+    ),
+    --3. Latest manager if more than 1 is attached
+    latest_manager AS (
+        SELECT
+            mine_no                         ,
+            MAX(contact_cid) AS contact_cid
+        FROM now_manager
+        WHERE mine_no||add_dt IN (
+            SELECT mine_no||add_dt
+            FROM latest_now
+        )
+        GROUP BY mine_no
+    ),
+    --4. Select existing manager record
+    existing_manager AS (
+        SELECT
+            mine_no                         ,
+            contact_cid AS person_combo_id  ,
+            mine_no||contact_cid AS mgr_combo_id
+        FROM latest_manager
+        where mine_no||contact_cid IN (
+            SELECT  mgr_combo_id
+            FROM    ETL_MANAGER
+        )
+    ),
+    --5. Check if manager is an existing person
+    existing_person AS (
+        SELECT DISTINCT ON (person_combo_id)
+            person_combo_id
+        FROM existing_manager
+        WHERE person_combo_id IN (
+            SELECT  person_combo_id
+            FROM    ETL_Manager
+        )
+    ),
+    --6. Extract contact info and formatting
+    existing_person_info AS(
+        SELECT
+            cid AS person_combo_id  ,
+            COALESCE(NULLIF(regexp_replace(contact_info.name,' ', '', 'g'),''),'Unknown') first_name,
+            COALESCE(NULLIF(regexp_replace(contact_info.l_name,' ', '', 'g'),''),'Unknown') surname ,
+            NULLIF(regexp_replace(phone, '\D', '','g'),'')::numeric phone_no                        , --Extract numbers only from phone_no field
+            COALESCE(NULLIF(regexp_replace(contact_info.email,' ', '', 'g'),''),'Unknown') email    ,
+            COALESCE(contact_info.add_dt,now()) effective_date
+        FROM mms.mmsccn contact_info
+        WHERE cid IN (
+            SELECT  person_combo_id
+            FROM    existing_person
+        )
+    ),
+    -- Remove the extra digit in phone number
+    existing_person_info_get_phone AS(
+        SELECT
+            gen_random_uuid() AS party_guid   ,
+            person_combo_id                   ,
+            first_name                        ,
+            surname                           ,
+            CASE
+                WHEN phone_no>=10^10 AND phone_no<2*10^10 THEN (phone_no-10^10 )::varchar--Remove country code for phone_no
```

Comment:
I'm wondering if country code or support for international numbers is something we actually need to have in our system. I could see some of the company contacts potentially having an overseas number for a parent company. OOS right now, I would think.